specification/_types/query_dsl/abstractions.ts

```diff
@@ -373,6 +374,13 @@ export class QueryContainer {
    * @doc_id query-dsl-text-expansion-query
    */
   text_expansion?: SingleKeyDictionary<Field, TextExpansionQuery>
+  /**
+   * Supports prototyping text_expansion query results by sending in precomputed tokens with the query. Not intended for production workflows.
+   * @availability stack since=8.13.0
```

Comment:
**Non-blocking question**
Whenever we mention **stack**, we mean stateful. Am I right?

Reply:
Correct!